feature/ledgermanual/src/main/java/com/moneymong/moneymong/ledgermanual/view/LedgerManualCategoryBottomSheet.kt

```diff
@@ -0,0 +1,301 @@
+                    )
+                }
+            }
+        }
+    }
+}
+
+@OptIn(ExperimentalLayoutApi::class)
+@Composable
+fun LedgerManualCategoryBottomSheetContent(
+    modifier: Modifier = Modifier,
+    categories: List<String>,
+    onDismissRequest: () -> Unit,
+    onClickCreate: () -> Unit,
+) {
+    Column(
+        modifier = modifier
+            .fillMaxWidth()
+            .height(448.dp)
+            .background(White)
+            .padding(horizontal = MMHorizontalSpacing, vertical = 20.dp),
+    ) {
+        Icon(
+            modifier = Modifier
+                .align(alignment = Alignment.End)
+                .noRippleClickable(onDismissRequest),
+            painter = painterResource(R.drawable.ic_close_default),
+            contentDescription = null
+        )
+        Spacer(modifier = Modifier.height(4.dp))
+        Row(
+            modifier = Modifier.fillMaxWidth(),
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.CenterVertically,
+        ) {
+            Text(
+                style = Heading4,
+                color = Gray10,
+                text = "카테고리",
+            )
+            Text(
+                modifier = Modifier.noRippleClickable(onClickCreate),
+                style = Body3,
+                color = Blue04,
+                text = "추가",
+            )
+        }
+        Spacer(modifier = Modifier.height(4.dp))
+        Text(
+            style = Body2,
+            color = Gray05,
+            text = "원하는 카테고리를 마음대로 만들 수 있어요",
+        )
+        Spacer(modifier = Modifier.height(16.dp))
+        FlowRow(
+            horizontalArrangement = Arrangement.spacedBy(12.dp),
+        ) {
+            categories.forEach {
+                MDSOutlineTag(
+                    text = it,
+                    iconResource = R.drawable.ic_close_default,
+                    onClick = {},
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun LedgerManualCategoryCreateBottomSheetContent(
+    modifier: Modifier = Modifier,
+    textFieldValue: TextFieldValue,
+    isSystemCategoryError: Boolean,
+    categories: List<String>, // TODO API response
+    onValueChange: (TextFieldValue) -> Unit,
+    onClickRegister: () -> Unit,
+    onPrev: () -> Unit,
+) {
+    val maxCount = 10
+    var isFilled by remember { mutableStateOf(false) }
+    val focusRequester = remember { FocusRequester() }
+    val keyboard = LocalSoftwareKeyboardController.current
+    val isExists = categories.contains(textFieldValue.text)
+    val helperText by remember(isSystemCategoryError, isExists) {
+        derivedStateOf {
+            when {
+                isSystemCategoryError -> "사용할 수 없는 카테고리 이름이에요"
+                isExists -> "이미 있는 카테고리에요"
+                else -> ""
+            }
+        }
+    }
+
+    BackHandler {
+        keyboard?.hide()
+        onPrev()
+    }
```

Comment:
키보드가 내려간 상태에서 시스템 백 버튼을 눌러야
`BackHandler` 의 람다 식이 호출되어 `keyboard?.hide()`가 없어도 동일하게 동작합니다.

```suggestion
    BackHandler {
        onPrev()
    }
```